app/scripts/lib/transaction/eip5792.ts

```diff
@@ -135,15 +135,15 @@ export async function getCapabilities(_address: Hex, _chainIds?: Hex[]) {
 function validateSendCalls(
   sendCalls: SendCalls,
   dappChainId: Hex,
-  disabledChains: Hex[],
+  disabledChainsAddresses: Record<Hex, Hex[]>,
```

Comment:
Minor, `disabledUpgradeAccountsByChain`?

Reply:
PR updated to address this